### DISCUSSION TITLE
Calendar quick create template lost

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#14648] [How to] [answered]
Dear all:

We have lost our calendar quick create template (right side panel). 

This is our view now: 

![tempsnip_2](https://user-images.githubusercontent.com/83352572/117584185-cbfc5380-b10b-11eb-8929-6c7936f9ac21.png)

And we have lost this right one: 

![tempsnip_3](https://user-images.githubusercontent.com/83352572/117584197-dc143300-b10b-11eb-84cc-5ae01a233d56.png)

The only thing that we have changed is the /config/modules/calendar.php in order to change 5 days to 7 in calendar view.

The event creation works fine but the issue is that we are unable to quick create from side panel.

Yetiversion: 6.1.0

Any idea to recover this?

Thanks in advance and best regards.

**Comment by rskrzypczak** [comment]
It is difficult to say something here, because I do not see the relationship between the elements described by you.
You had to change something else.
First, check if you have changed the file encoding by modifying the $HIDDEN_DAYS_IN_CALENDAR_VIEW variable.

**Comment by sistemasnow** [comment, marked as answer]
The issue was solved.

Configuration --> Standard modules --> Edit fields --> select Calendar --> the view was in advanced mode --> changed it to basic mode and the calendar works fine again.

Thanks in advance and regards.
